src/Cake.Common/Tools/ReportGenerator/ReportType.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Represents ReportGenerator's output formats
+    /// </summary>
+    public enum ReportType
```

Comment:
As per a recent conversation with @devlead on one of my Pull Requests, I think that this should be ReportGeneratorReportType:

https://github.com/cake-build/cake/pull/461#discussion-diff-42491591